src/utils/valid_headers_only.js

```diff
@@ -0,0 +1,46 @@
+const { validateHeaderName, validateHeaderValue } = require('http');
+const { isHopByHopHeader } = require('./is_hop_by_hop_header');
+
+/**
+ * Filters out invalid headers.
+ * @param {string[]} array
+ * @returns Filtered headers.
+ */
+const validHeadersOnly = (array) => {
+    const rawHeaders = [];
```

Comment:
Those moves to a separate file could've probably been done in a separate PR. It's hard to see if there are any differences in the function code this way.

Reply:
[`f04773e` (#157)](https://github.com/apify/proxy-chain/pull/157/commits/f04773e287a9ac2105375f389694d2bde596849b)

Reply:
that is a separate commit, not separate PR. our point is that you do many changes in a single PR, and moving files around basically disables diffing, we can't easily tell whether you just moved them or also changed something in the implementation. 

same as with the linting problems

Reply:
Yup, what @B4nan says. I have to read the whole functions to see if they're the same or not. And even then I'm not really sure because the code is not side-by-side. It's a quality-of-reviewer's-life suggestion :)

Reply:
I agree, though it's not much noise given those are separate test files. If a PR is big, it's good to review it on a per-commit basis.